src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure {
+
+    protected double area;
+    protected Color color;
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public abstract void setArea();
+
+    public abstract void getInfo();
```

Comment:
let's move this method to an interface, they are usually used to declare a behavior

Reply:
fixed